packages/docs-components/src/knobs/useLogKnob.ts

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { LogContext } from './KnobContexts'
+
+const defaultFormatter = (name: string) => `${new Date().toLocaleTimeString()}: ${name}`
+
+const useLogKnob = <T = (...args: any[]) => any>(
+  name: string,
+  callback?: T,
+  formatter: Function = defaultFormatter,
```

Comment:
Can we specify the `formatter` type, instead of using a general `Function`? For instance, it would be nice to specify that the function should return `string`.

Reply:
Done 👍